test/fuzzer/fuzzer.cpp

```diff
@@ -371,6 +370,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t data_size) noe
             ASSERT(std::equal(ref_host.recorded_logs.begin(), ref_host.recorded_logs.end(),
                 host.recorded_logs.begin(), host.recorded_logs.end()));
 
+            ASSERT_EQ(ref_host.recorded_blockhashes.size(), host.recorded_blockhashes.size());
```

Comment:
This is checked by `std::equal` is it just for simpler failure messsage?

Reply:
This is redundant check for easier debugging.